application/views/admin/authentication/login.php

```diff
@@ -107,6 +107,9 @@
                                     'name' => 'loginlang',
                                     'data' => $languageData,
                                     'value' => $language,
+                                    'events' => array(
+                                        'change' => 'function() {this.form.submit();}'
```

Comment:
And for other plugin ?
And it's better if it can be easily remplaced by plugin OR Admin theme.

It must be in external js file.